sql/moz-fx-data-shared-prod/telemetry_derived/active_users_aggregates_device_v1/query.sql

```diff
@@ -19,6 +19,16 @@ SELECT
   COUNT(DISTINCT IF(days_since_seen = 0, client_id, NULL)) AS dau,
   COUNT(DISTINCT IF(days_since_seen < 7, client_id, NULL)) AS wau,
   COUNT(DISTINCT client_id) AS mau,
+  COUNT(
+    DISTINCT IF(
+      days_since_seen = 0
+      AND normalized_app_name = 'Firefox Desktop'
+      AND active_hours_sum > 0
+      AND uri_count > 0,
+      client_id,
+      NULL
+    )
+  ) AS qdau_desktop,
```

Comment:
It looks like this is only for desktop users.  Since mobile is also part of the [user story](https://docs.google.com/document/d/1ubnHZBiTTHDnqi8lYdm_qztONnoKiFEPFYNeOLfbRdI/edit), would it need to be addressed as well?

Reply:
For mobile DAU = QDAU, so there is no need to replicate the numbers in a new column, but it's definitely worth a  clarification of the column description, added!